spec/hipchat_api_v1_spec.rb

```diff
@@ -48,7 +48,7 @@
         OpenStruct.new(:code => 403)
       }
 
-      expect { room.history }.to raise_error(HipChat::UnknownResponseCode)
+      expect { room.history }.to raise_error(HipChat::Unauthorized)
```

Comment:
Doesn't this change the contract of the Gem? Does Unauthorized extend from UnknownResponseCode?